toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,578 @@
+        }).text(label.label));
+
+        row.append($("<td>").attr("class", "blue " + totalImageClass).text(label.totalImages));
+        row.append($("<td>").attr("class", "blue").text(label.matchedTop1Per.toFixed(2)));
+        row.append($("<td>").attr("class", "blue").text(label.matchedTop5Per.toFixed(2)));
+        row.append($("<td>").attr("class", "blue").text(label.match1));
+        row.append($("<td>").attr("class", "blue").text(label.match2));
+        row.append($("<td>").attr("class", "blue").text(label.match3));
+        row.append($("<td>").attr("class", "blue").text(label.match4));
+        row.append($("<td>").attr("class", "blue").text(label.match4));
+        row.append($("<td>").attr({
+            "class": "blue filter-image " + misclassifiedClass,
+            "data-id": label.id,
+            "data-type": "misclassified"
+        }).text(label.misclassifiedTop1));
+
+        row.append($("<td>").html("<input id=\"id_\"" + id + " name=\"id[" + id + "]\" type=\"checkbox\" value=\"" + id + "\" onClick=\"highlightRow(this);\"></input>"));
+        myTableBody.append(row);
+    });
+
+
+}
+
+// eslint-disable-next-line no-unused-vars
+function filterLabelTable(rowNum, Datavar) {
+    var labelSummaryFiltered = [];
+    var allEmpty = true;
+    // eslint-disable-next-line no-unused-vars
+    $("input[id ^= \"fl_\"]").each(function (element) {
+        var property = this.id.split("_")[1];
+        var value = this.value;
+        var compare = this.getAttribute("data-compare");
+        if (value) {
+            allEmpty = false;
+
+            labelSummary.forEach(function (label) {
+                var compareResult = false;
+
+                if (compare === "contains") {
+                    compareResult = label[property].toLowerCase().includes(value.toLowerCase());
+
+                } else {
+                    compareResult = label[property] == value;
+                }
+                if (compareResult) {
+                    labelSummaryFiltered.push(label);
+                }
+            });
+        }
+    });
+    if (allEmpty) {
+        loadLabelSummary(labelSummary);
+    } else {
+        loadLabelSummary(labelSummaryFiltered);
+    }
+}
+
+// eslint-disable-next-line no-unused-vars
+function clearLabelFilter() {
+    $("input[id ^= \"fl_\"]").val("");
+
+    loadLabelSummary(labelSummary);
+}
+
+// eslint-disable-next-line no-unused-vars
+function highlightRow(element) {
+    var parentRow = element.parentElement.parentElement;
+    parentRow.classList.toggle("highlight-row");
+    console.log(element.parentElement.parentElement);
+
+}
+
+
+//Create copy of imageSummary
+var imageSummaryFiltered = JSON.parse(JSON.stringify(imageSummary));
+
+// eslint-disable-next-line no-unused-vars
+function clearResultFilter() {
+    $("input[id ^= \"fli_\"]").val("");
+    imageSummaryFiltered = JSON.parse(JSON.stringify(imageSummary));
+    loadImageResults(imageSummaryFiltered);
+    updateItems();
```

Comment:
Codacy found an issue: ['updateItems' was used before it was defined.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)